packages/desktop/components/popups/send/SendNftForm.svelte

```diff
@@ -0,0 +1,178 @@
+<script lang="ts">
+    import { onMount } from 'svelte'
+    import { get } from 'svelte/store'
+    import { Button, Error, NetworkInput, NftInput, OptionalInput, RecipientInput } from '@ui'
+    import { closePopup, openPopup, PopupId } from '@auxiliary/popup'
+    import { localize } from '@core/i18n'
+    import { selectedAccount } from '@core/account/stores'
+    import { Output } from '@iota/sdk/out/types'
+    import {
+        getDefaultTransactionOptions,
+        getOutputParameters,
+        NewNftTransactionDetails,
+        newTransactionDetails,
+        NewTransactionType,
+        setNewTransactionDetails,
+    } from '@core/wallet'
+    import features from '@features/features'
+    import { activeProfile } from '@core/profile'
+    import { prepareOutput } from '@core/account'
+    import { handleError } from '@core/error/handlers'
+    import { MAX_METADATA_BYTES, MAX_TAG_BYTES } from '@core/utils'
+    import { OptionalInputType } from '@core/wallet/utils/send/sendUtils'
+
+    export let validateInputs: (
+        inputValidations?: any,
+        optionalInputValidations?: {
+            validate?: (promise: Promise<unknown>) => Promise<void>
+            value: string
+            optionalInputType: OptionalInputType
+            byteLimit: number
+            errorMessage: string
+        }[]
+    ) => Promise<boolean> = () => Promise.resolve(true)
+    export let isTransferring: boolean = false
+
+    const transactionDetails = get(newTransactionDetails)
+    const { expirationDate, layer2Parameters, disableAssetSelection } = transactionDetails as NewNftTransactionDetails
+
+    let { nftId, recipient, tag, metadata, giftStorageDeposit, disableChangeExpiration, disableToggleGift } =
+        transactionDetails as NewNftTransactionDetails
+
+    let preparedOutput: Output
+    let isPreparingOutput = false
+
+    let nftInput: NftInput
```

Comment:
When Send form is opened from the "Homepage" and than user switches to Nft tab there are no nfts in the input dropdown. Mybe we can add a select dropdown with list of available Nfts like we do with recipients and destination network inputs.
![image](https://github.com/iotaledger/firefly/assets/52735957/d80775f7-7291-4ebf-9ad2-a5b45ae98cd6)

Reply:
Are you sure you have NFTs on this profile ? 😅 
I think that it worked for me.